---
title: The Role of Gene Duplication in Mediating Peto's Paradox in Afrotheria and Chiroptera
event: Thesis Defense
event_url: ""

location: Zoom
# address:
#   street: 97 Summit Road
#   city: Newry
#   region: MR
#   postcode: ''
#   country: United States

summary: |
  Based on emperical studies of humans, mice, and various other species, an individual's cancer risk is directly proportional to their cell count (body size) and lifespan. This leads to a theoretical prediction that large and/or long-lived species would possess a higher predisposition to cancer compared to smaller, shorter-lived species; compounding this risk is the fact that body size and lifespan are strongly correlated. However, in a phenomenon known as Peto's Paradox, cancer risk between species does not correlate with either their body sizes or lifespans. This implies that enhanced cancer resistance mechanisms must co-evolve with increases in body size and lifespan; however, there are many ways this can come about. Rather than reinventing the wheel, species can carry an increased load of cancer risk by increasing the number of wheels they have. My thesis focuses on the role tumor suppressor gene duplications play in Peto's Paradox: Chapter 1 explores whether or not tumor suppressor genes are especially enriched among duplicated genes in large, long lived species, while Chapters 2 and 3 functionally characterize two such duplications. Overall, my work here highlights the vital role that tumor suppressor gene duplicates play in lowering the cancer risk of large, long-lived species, while also highlighting new questions for future work, especially regarding antagonistic pleitropy and growth-suppression paradoxes with these duplicates. 

abstract: |
  Cancer is a disease intrinsic to multicellularity. Within a species, body size and cancer risk are strongly correlated with cancer risk; between species, however, this correlation no longer holds. This phenomena, known as Peto’s Paradox, requires that species evolve cancer suppression mechanisms alongside increases in size and lifespan. Previous candidate gene studies looking at tumor suppressor duplications in large, long-lived species point towards a greater role of gene duplication in resolving Peto’s Paradox. Thus, in this thesis, I characterized gene duplications genome-wide in publicly-available genomes to determine if tumor suppressor genes in particular were enriched among duplicated genes in large and/or long-lived species. Then, to explore the functional consequences of these duplicates, I selected two hits in large, long-lived species - *LIF* in the African Savanna Elephant (*Loxodonta africana*) and *TP53* in the Little Brown Bat (*Myotis lucifugus*) to characterize in primary fibroblasts, and determine their effects on cell cycle and cell death in response to stress. 
  Expanding upon previous candidate-gene-based surveys of gene duplication in Elephants, we used a Reciprocal Best-Hit BLAT strategy to obtain copy numbers of all protein-coding genes in *Atlantogenatan* genomes to see if there is any correlation between the copy number of duplicates and changes in body size along the phylogenetic tree. From an initial set of 18,011 protein-coding genes in hg38, we identified a median of 13,880 genes in *Atlantogenatan* genomes, of which a median of 940 genes are duplicated. We find that, just as body size fluctuates throughout *Atlantogenata*, genes involved in tumor suppressor pathways are also duplicated throughout the phylogenetic tree. Extant species of elephants, however, show active transcription of both canonical and duplicated copies of tummor suppressors that duplicated prior to and during their sudden increase in body size, suggesting that the duplication of tumor suppressor genes facilitates the evolution of increased body size by compensating for the increased cancer risk.
  The resurrection and re-functionalization of a leukemia inhibitory factor pseudogene (*LIF6*) with pro-apoptotic functions in elephants and their extinct relatives (*Proboscideans*) may have played a role in resolving Peto’s Paradox. *LIF6* is transcriptionally up-regulated by *TP53* in response to DNA damage, and translocates to the mitochondria where it induces apoptosis. Phylogenetic analyses of living and extinct *Proboscidean* *LIF6* genes indicates its *TP53* response element evolved coincident with the evolution of large body sizes in the *Proboscidean* stem-lineage. These results suggest that re-functionalizing of a pro-apoptotic *LIF* pseudogene may have been permissive (though not sufficient) for the evolution of large body sizes in *Proboscideans*.
  In the long-lived bat, *Myotis lucifugus*, I describe the only known full-locus duplication of *TP53* in nature, which may play a role in shaping its unique stress response. This bat has 8 copies of *TP53*, a central regulator of the DNA damage response present in all living organisms. A syntenic duplication of *TP53*-WRAP53 has occurred within its genome, leading to two copies of *TP53* that have conserved both regulatory and transcriptional functionality. Relative to 4 other closely related bat species (*M. evotis*, *M. thysanodes*, *M. yumanensis*, and *E. fuscus*), the Little brown bat demonstrates a unique resistance to both DNA damage as well as generalized oxidative stress, which matches the phenotypic changes induced by a transgenic *TP53* locus duplication in a previously-described mouse model. 
  Overall, these results suggest that gene duplication plays an important role in Peto’s Paradox; while individual cases of tumor suppression duplication can facilitate the evolution of increased lifespans and body sizes, the suppression of cancer risk likely requires a concert of genetic changes. This fits in with a polygenic or omnigenic model of cancer resistance and tumorigenesis, and suggests that large-scale integrated functional genomic approaches to studying Peto’s Paradox may provide greater clarity into how large, long-lived species suppress their increased cancer risk.


# Talk start and end times.
#   End time can optionally be hidden by prefixing the line with `#`.
date: "2020-06-12T11:00:00Z"
#date_end: "2030-06-01T15:00:00Z"
all_day: false

# Schedule page publish date (NOT talk date).
publishDate: "2020-06-23T00:00:00Z"

authors: ["JM Vazquez"]
tags: ["Petos-paradox", "Gene-Duplication", "LIF", "TP53", "Bats", "Elephants", "Thesis"]


# Is this a featured talk? (true/false)
featured: true

image:
  caption: ''
  focal_point: ""

# links:
# - icon: twitter
#   icon_pack: fab
#   name: Follow
#   url: https://twitter.com/georgecushen
url_code: ""
url_pdf: ""
url_slides: "JuanVazquezThesisDefense.pptx"
url_video: "JuanVazquezThesisDefense.mp4"

# Markdown Slides (optional).
#   Associate this talk with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides: ""

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects:
- ["Petos-paradox"]

# Enable math on this page?
math: true
---
